src/Raven.Client/Documents/Operations/Replication/PullReplicationAsSink.cs

```diff
@@ -77,6 +78,7 @@ public override DynamicJsonValue ToJson()
             djv[nameof(AllowedHubToSinkPaths)] = AllowedHubToSinkPaths;
             djv[nameof(AllowedSinkToHubPaths)] = AllowedSinkToHubPaths;
             djv[nameof(AccessName)] = AccessName;
+            djv[nameof(Url)] = Url;
```

Comment:
didn't we write it in `var djv = base.ToJson();` ?

Reply:
right! done.